backend/src/ee/services/dynamic-secret/providers/models.ts

```diff
@@ -100,6 +117,27 @@ export const DynamicSecretSqlDBSchema = z.object({
   database: z.string().trim(),
   username: z.string().trim(),
   password: z.string().trim(),
+  passwordRequirements: z
+    .object({
+      minLength: z.number().min(1).max(100),
+      maxLength: z.number().min(1).max(100),
```

Comment:
why do we support ranges? curious if just a single length value could make sense here

Reply:
removed